### PR TITLE
feat: paginate connectors list

### PR DESCRIPTION
**Pull request by sumitd94**
## Description
Added pagination to the list of connectors we are showing

Screenshot 2024-03-05 at 10 33 35 PM

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would impact existing functionality)
- [ ] Styling change
- [ ] Documentation update
- [ ] Refactoring
- [ ] Chore
  
## Checklist:
- [x] Ensure a branch name is prefixed with `feature`, `bugfix`, `hotfix`, `release`, `style` or `chore` followed by `/` and branch name e.g `feature/add-salesforce-connector`
- [ ] Added unit tests for the changes made (if required)
- [x] Have you made sure the commit messages meets the guidelines?
- [x] Added relevant screenshots for the changes
- [x] Have you tested the changes on local/staging?
- [x] Have you made sure the code you have written follows the best practises to the best of your knowledge?

### CODE REVIEW

File: src/views/Dashboard/ListConnectors.tsx [hunk 1]
```diff
@@ -11,6 +11,11 @@ import {
 } from '@chakra-ui/react';
 import EntityItem from '@/components/EntityItem';
 import { ConnectorItem } from '../Connectors/types';
```
Comment (macintushar) [inline src/views/Dashboard/ListConnectors.tsx:13]: Imports should be using `@` and not be absolute like `../../`

File: src/views/Dashboard/Pagination.tsx
Comment (macintushar) [inline src/views/Dashboard/Pagination.tsx:1]: NIT :  file can be in its own directory like `/Dashboard/Pagination/Pagination.tsx `

File: src/views/Dashboard/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    justifyContent='center'
+    alignItems='center'
+    backgroundColor='gray.300'
+    onClick={handlePageChange}
+    minWidth='0'
+    padding={0}
+    _hover={{ backgroundColor: 'gray.300' }}
+  >
+    {type === PAGE_CHANGE_BUTTON_TYPE.PREVIOUS ? (
+      <MdKeyboardArrowLeft color='#98A2B3' />
```
Comment (macintushar) [inline src/views/Dashboard/Pagination.tsx:34]: Are the color's used in this not already available in our custom color tokens? If not then they should be updated in the design to use our existing color tokens, unless it's a new color

**Review by macintushar — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Pagination.tsx (hunk 2 above)
Comment (sumitd94) [reply src/views/Dashboard/Pagination.tsx:34]: the icons do not support tokens unfortunately!

File: src/views/Dashboard/ListConnectors.tsx (hunk 1 above)
Comment (sumitd94) [reply src/views/Dashboard/ListConnectors.tsx:13]: Will push this in a different PR, not part of this PR change :P

**Pull request merged**